src/workspace-hack/Cargo.toml

```diff
@@ -79,6 +79,7 @@ reqwest = { version = "0.11", features = ["__tls", "default-tls", "hyper-tls", "
 ring = { version = "0.16", features = ["alloc", "dev_urandom_fallback", "once_cell", "std"] }
 scopeguard = { version = "1", features = ["use_std"] }
 serde = { version = "1", features = ["alloc", "derive", "rc", "serde_derive", "std"] }
+simd-json = { version = "0.7", features = ["allow-non-simd", "serde", "serde_impl", "serde_json", "swar-number-parsing"] }
```

Comment:
what's the purpose of this?

Reply:
I'm just wondering why we now need a lot of features that we originally don't need?

Reply:
strange to me. I removed `allow-non-simd` feature in source's cargo.toml but it appears here

Reply:
It is generated by workspace hack. I tried to run `cargo hakari generate` but nothing change